redis/client.go

```diff
@@ -2,7 +2,6 @@ package redis
 
 import (
 	"fmt"
-
```

Comment:
I think both of these changes could be removed - since we didn't have to touch this file.